pandera/strategies.py

```diff
@@ -978,12 +978,6 @@ def make_row_strategy(col, checks):
     if index is not None:
         strategy = set_pandas_index(strategy, index.strategy(size=size))
 
-    strategy = strategy.map(  # type: ignore
```

Comment:
for the record, this piece of code was redundant: `col_dtypes` already overrides the column-level datatype with the dataframe-level datatype.